include/Clip.h

```diff
@@ -110,6 +110,22 @@ class LMMS_EXPORT Clip : public Model, public JournallingObject
 		return m_autoResize;
 	}
 
+	/*! \brief Set whether a clip has been resized yet by the user or the knife tool.
+	 *
+	 *  If a clip has been resized previously, it will not automatically 
+	 *  resize when editing it.
+	 *
+	 */
+	inline void setHasBeenResized( const bool r )
+	{
+		m_hasBeenResized = r;
+	}
+
+	inline const bool getHasBeenResized() const
+	{
+		return m_hasBeenResized;
+	}
```

Comment:
```suggestion
	void setHasBeenResized(const bool r)
	{
		m_hasBeenResized = r;
	}

	bool getHasBeenResized() const
	{
		return m_hasBeenResized;
	}
```
Style, removed `inline`, removed `const` from return type